index.js

```diff
@@ -348,16 +348,17 @@ const getCiBuildId = async () => {
       }
     )
 
-    if (runsList && runsList.data) {
-      // Use the total_count, every time a job is restarted the list has
-      // the number of jobs including current run and previous runs, every time
-      // it appends the result.
-      core.debug(
-        `fetched run list with ${runsList.data.total_count} records`
-      )
-      parallelId = `${GITHUB_RUN_ID}-${runsList.data.total_count}`
+    if (
+      runsList &&
+      runsList.data &&
+      runsList.data.jobs &&
+      runsList.data.jobs.length
+    ) {
+      const jobId = runsList.data.jobs[0].id
```

Comment:
Can you please explain what these job records and IDs would be? For example, on the first parallel run, the list will be empty, right? 

When the user re-runs the action, what is the job[0]? If the user re-runs the job again, is the job[0] still the same as the first re-run? I think just a good JSDoc comment here is crucial

Reply:
Sure thing! will add it so, we can have it much clear!